maintainers/maintainer-list.nix

```diff
@@ -6516,6 +6621,7 @@
   };
   lschuermann = {
     email = "leon.git@is.currently.online";
+    matrix = "@leon:is.currently.online";
```

Comment:
```suggestion
    matrix = "@leons:is.currently.online";
```

So close. :)

Reply:
I had to manually type some ;) Thank you.